WeakAurasOptions/WeakAurasOptions.lua

```diff
@@ -4072,6 +4072,84 @@ function WeakAuras.PickDisplayMultiple(id)
   frame:PickDisplayMultiple(id);
 end
 
+function WeakAuras.PickDisplayMultipleShift(target)
+  if (frame.pickedDisplay) then
+    -- get first aura selected
+    local first;
+    if (WeakAuras.IsPickedMultiple()) then
+      first = tempGroup.controlledChildren[1];
+    else
+      first = frame.pickedDisplay;
+    end
+    if (first and first ~= target) then
+      -- check that id is in same group as first
+      local firstData = WeakAuras.GetData(first);
+      -- in a group
+      if (firstData.parent) then
+        local sameGroup = false;
+        local group = WeakAuras.GetData(firstData.parent);
+        for index, child in pairs(group.controlledChildren) do
```

Comment:
You can use GetData(target) here and check .parent of that.

Reply:
I could do 
      local firstData = WeakAuras.GetData(first);
      local targetData = WeakAuras.GetData(target);
and then compare firstData.parent == targetData.parent (which is more readable and concise)
But after checking both items are in the same group, i still need to iterate over the group children, so another GetData() on the parent

The way i did it save a GetData() call, i have not checked the complexity of this call, do you think their is not enough performance gain doing what i did for what we loose in readability ?

Reply:
GetData is cheap, iterating over all childrens is not as cheap. So using that to figure out whether they are the same parent is probably better.